lib/open-amp/resource_table.h

```diff
@@ -16,13 +16,20 @@ extern "C" {
 
 #if (CONFIG_OPENAMP_RSC_TABLE_NUM_RPMSG_BUFF > 0)
 
-#define VDEV_ID                 0xFF
+#define VDEV_ID 0xFF
+#ifdef CONFIG_IPM_RX_CHANNEL_ID
```

Comment:
you modified VDEV_ID line. Please don't do that here as it is not the scope of this patch. 

Your patch should contain only the changes that are described in the commit message. Any other collateral changes can be done in separate patches with proper explanation.